src/analytics/lambdas/redshift-data.ts

```diff
@@ -20,6 +21,10 @@ import { ExistingRedshiftServerlessCustomProps, ProvisionedRedshiftProps } from
 export function getRedshiftClient(roleArn: string) {
   return new RedshiftDataClient({
     ...aws_sdk_client_common_config,
+    requestHandler: new NodeHttpHandler({
+      connectionTimeout: 50000,
```

Comment:
It does not make sense to change the connection timeout.

Reply:
changed connectionTimeout to 5000ms.